zbus_macros/src/proxy.rs

```diff
@@ -717,12 +717,13 @@ fn gen_proxy_property(
         let receive_method = match emits_changed_signal {
             PropertyEmitsChangedSignal::True | PropertyEmitsChangedSignal::Invalidates => {
                 let (_, ty_generics, where_clause) = m.sig.generics.split_for_impl();
-                let receive = format_ident!("receive_{}_changed", method_name);
+                let receive = format_ident!("receive_{method_name}_changed");
                 let gen_doc = format!(
                     "Create a stream for the `{property_name}` property changes. \
-                This is a convenient wrapper around [`{proxy_name}::receive_property_changed`]."
+                     This is a convenient wrapper around [`{proxy_name}::receive_property_changed`]."
                 );
                 quote! {
+                    #(#other_attrs)*
```

Comment:
That easy? :laughing: Would be good to add a test case (doesn't have to be a new separate one but a part of an existing one).

Reply:
hmm.. good question. Nevermind.